github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -807,7 +807,7 @@ presubmits:
           automation/test.sh
         env:
         - name: FEATURE_GATES
-          value: NonRootExperimental
+          value: NonRoot
         - name: NONROOT
```

Comment:
nit: didn't look to closely but shouldn't this be KUBEVIRT_NONROOT?
```suggestion
        - name: KUBEVIRT_NONROOT
```

Reply:
I agree but we need to first change kubevirt repo.

Reply:
I don't get it? Did I overlook that there's support for the `NONROOT` parameter somewhere in kubevirt/kubevirt? I looked at kubevirt/kubevirt automation/test.sh and didn't find anything?

Reply:
:man_facepalming:  You are right!

Reply:
Addressed in new commit